src/MobileList.js

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect } from 'react';
+import moment from 'moment';
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+require('twix');
+
+const MAX_DAYS_IN_LIST = 1;
+
+function EventTimes(props) {
+  const { rawTimes } = props;
+  let sortedTimesByDate = groupBy(sortBy(rawTimes,
```

Comment:
Are you familiar with the chained syntax for lodash? Would make this cleaner to read since the functions would be in order instead of nested

Reply:
cc: @john-osullivan 

Another item here, John, if you could just speak to this.